assets/css/sensei-course-theme/quiz.scss

```diff
@@ -7,8 +7,30 @@ $tablet-breakpoint: 768px;
 
 .sensei-course-theme__quiz {
 	&__main-content {
-		padding: 32px;
 		margin-bottom: 100px !important;
+		padding-top: clamp(2.5rem, 1.346rem + 3.846vw, 3.75rem);
+
+		.wp-block-post-title {
+			margin-top: clamp(1.688rem, 0.534rem + 3.846vw, 2.938rem);
+		}
+
+		.sensei-lms-quiz-back-to-lesson {
+			display: flex;
+			align-items: center;
+
+			svg {
+				width: 24px;
+				height: 24px;
+			}
+
+			/* The gap between the arrow and the text for mobile and smaller screens */
+			gap: 8px;
+
+			@media (min-width: 500px) {
```

Comment:
Same thing here re: breakpoint.

Reply:
Updated it here https://github.com/Automattic/sensei/pull/7093/commits/a2d416b0112f8cf773fd454e3414c5f48c12227f